tests/matmul/requirements.txt

```diff
@@ -1,3 +1,4 @@
 PyYAML>=5.4.1
 requests>=2.28.0
-enum_tools==0.6.4
\ No newline at end of file
+enum_tools==0.6.4
+numpy<2
```

Comment:
Why less than 2, that's a bit annoying, can we rather change our code to make it forward compatible?

Reply:
numpy 2 introduced [breaking changes](https://numpy.org/devdocs/numpy_2_0_migration_guide.html) - it was their first version bump in like 9 years? I don't know I'm sure we upgrade/make changes but for right now I'm doing `numpy<2` everywhere (and then some years from now I'll figure out numpy 2).